helm-charts/splunk-otel-collector/values.yaml

```
@@ -211,9 +211,48 @@ extraAttributes:
 agent:
   enabled: true
 
-  # This Flag enables k8s control plane metric collection.
-  # Details about control plane monitoring and related configurations are located in docs/advanced-configuration.md
-  controlPlaneEnabled: true
+  # Metric collection from k8s control plane components.
+  # For control plane configuration details see: docs/advanced-configuration.md#control-plane-metrics
+  controlPlaneMetrics:
+    apiserver:
+      # Specifies whether to collect apiserver metrics.
+      enabled: true
+    controllerManager:
+      # Specifies whether to collect controller manager metrics.
+      enabled: true
+    coredns:
+      # Specifies whether to collect coredns metrics.
+      enabled: true
+    etcd:
+      # Specifies whether to collect etcd metrics.
+      # For set up etcd metrics details see: docs/advanced-configuration.md#setting-up-etcd-metrics
+      enabled: false
+      secret:
+        # If name is empty the default name is used.
+        name: ""
+        # Option for creating a new secret or using an existing one.
+        create: false
+        # Used when secret.create=true creates a se. The PEM-format CA certificate for the etcd client.
```

```suggestion
        # Used when secret.create=false. The PEM-format CA certificate for the etcd client.
```

and the same for others

---

I think we want to keep this "secret.create=true", but I did get rid of everything else. I also added more comments for context. Let me know if you want this to still change.

---

Makes sense. I just saw create=false in the values.yaml before